examples/deadline/All-In-AWS-Infrastructure-Basic/python/setup.py

```diff
@@ -17,8 +17,8 @@
     packages=setuptools.find_packages(where="package"),
 
     install_requires=[
-        "aws-cdk.core==1.57.0",
-        "aws-rfdk==0.16.0"
+        "aws-cdk.core==1.61.1",
+        "aws-rfdk==0.17.0"
```

Comment:
Not sure about this....

Version 0.17.0 of RFDK uses CDK 1.57.0, right?

Reply:
Matches the Typescript version of the example. Hrm... that's a pickle. I guess we'll have to leave the examples on the last release, and bump them (including CDK version) on release.